app/src/main/java/com/framgia/soundclound/screen/local/TrackLocalAdapter.java

```diff
@@ -0,0 +1,77 @@
+package com.framgia.soundclound.screen.local;
+
+import android.databinding.DataBindingUtil;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+
+import com.framgia.soundclound.R;
+import com.framgia.soundclound.data.model.Track;
+import com.framgia.soundclound.databinding.ItemTrackLocalBinding;
+import com.framgia.soundclound.screen.local.TrackClickListener;
```

Comment:
WARNING: [ImportFromSamePackage] No need to import a type that lives in the same package
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#ImportFromSamePackage